synapse/cortex.py

```diff
@@ -5754,10 +5756,14 @@ async def _getMirrorProxy(self):
                 # we are part of the pool and were selected. Convert to local use.
                 return None
 
+            await s_common.wait_for(proxy.getNexsIndx(), timeout)
```

Comment:
since we're retrieving this, should we check that it's within some sane range of current? should we loop to try to get another if this one times out?

Reply:
updated